### PR TITLE
feat(ui): Toggle button states depending on Robot Capabilities

### PR DESCRIPTION
**Pull request by FrankZZ**
Description: 
This PR entails toggling the button states based on the capabilities reported by the Valetudo backend in:
- Home page
- Settings page

I also encountered some minor cosmetic things that I included in this PR such as:
- Add `dBm` unit to the WiFi signal settings field
- Capitalize `WiFi Status`

### CODE REVIEW

File: client/settings.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        const robotCapabilities = await ApiService.getCapabilities() || [];
+        const buttonStateMap = {
+            info: true,
+            timers: robotCapabilities.includes("DoNotDisturbCapability"),
+            "carpet-mode": robotCapabilities.includes("CarpetModeControlCapability"),
+            "cleaning-history": false, // commented out in settings.html?
+            "persistent-data": robotCapabilities.includes("PersistentMapControlCapability"),
+            consumables: robotCapabilities.includes("ConsumableMonitoringCapability"),
+            wifi: robotCapabilities.includes("WifiConfigurationCapability"),
+            mqtt: robotCapabilities.includes("WifiConfigurationCapability"),
```
Comment (Hypfer) [inline client/settings.js:15]: This can be true all the time

File: client/settings.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            token: false, // commented out in settings.html?
+            sound: robotCapabilities.includes("SpeakerVolumeControlCapability"),
+            "access-control": robotCapabilities.includes("WifiConfigurationCapability")
```
Comment (Hypfer) [inline client/settings.js:18]: Same for access-control

**Comment by Hypfer** [comment]
Hiding elements by default and only unhiding them if they're actually available would be even better.

Now, I see stuff disappearing when I open the settings page

**Comment by FrankZZ** [comment]
> Hiding elements by default and only unhiding them if they're actually available would be even better.
> 
> Now, I see stuff disappearing when I open the settings page

I've addressed this in #774

**Pull request merged**